packages/number-field/src/NumberField.ts

```diff
@@ -400,7 +405,11 @@ export class NumberField extends TextfieldBase {
             }
         }
         this.value = value;
```

Comment:
This line will trigger the setter. Will the following lines _never_ accidentally dispatch a second `change` event?

Reply:
no because in the setter we are updated the lastCommittedValue which will cause the check on the line 409 to fail making sure there is not second `change` event